<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RigWidget</class>
 <widget class="QWidget" name="RigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>86</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelRig">
     <property name="text">
      <string>Rig:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxRig"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonBrowseRigPresets">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelLensType">
     <property name="text">
      <string>Lens type:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxLensType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelHfov">
     <property name="toolTip">
      <string>Input lenses' horizontal Field of View</string>
     </property>
     <property name="text">
      <string>FOV:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxHfov">
     <property name="specialValueText">
      <string>auto</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
